mdata/cache/cache.go

```diff
@@ -0,0 +1,117 @@
+package cache
+
+import (
+	"flag"
+	"sync"
+
+	accnt "github.com/raintank/metrictank/mdata/cache/accnt"
```

Comment:
fixed that. moved them out of the `cache` package previously and forgot to update the `package` declaration